### ISSUE TITLE
Colors not visible in phones

### ISSUE THREAD

**Issue by satyabrata-dash5** [dilipchandima/rn-select-date-range#18]
on OPPO phones, this was looking like this.
please add some props to it.
Thank you
![screenshot-nimbusweb me-2023 02 22-12_18_11](https://user-images.githubusercontent.com/57708184/220586679-35a86a78-1da9-432c-9cd9-33f995013181.png)

**Comment by oman21** [comment]
yeah it because dark mode and there is no props for styling year, date and day name. please ada styling for those @dilipchandima

**Comment by avantmarketing1** [comment]
I have the same problem on my android phone. The css can be edited in: /rn-select-date-range/lib/cjs/DateRangePicker.js

**Comment by ILoomans** [comment]
I created a PR for this [here](https://github.com/dilipchandima/rn-select-date-range/pull/19)
